code/modules/projectiles/guns/projectile/automatic/solmarine.dm

```diff
@@ -90,7 +90,7 @@
 
 	shotgun_haver = FALSE
 
-/obj/item/gun/projectile/automatic/omnirifle/omnicarbine/solmarine/shotgunless/sawn
+/obj/item/gun/projectile/automatic/omnirifle/omnicarbine/solmarine/shotgunless_sawn
```

Comment:
no reason to do this, a shotgunless solmarine rifle that isnt sawnoff might be made ion the future

Reply:
Yes their is

Reply:
what reason then

Reply:
Gun transmutations